com.unity.shadergraph/Editor/ShaderFoundry/Linker/Properties/PropertyDeclarations.cs

```diff
@@ -0,0 +1,382 @@
+using System;
+using System.Collections.Generic;
+
+namespace UnityEditor.ShaderFoundry
+{
+    class PropertyDeclarations
```

Comment:
Declare this as a static class (makes it easy to know that it is not a usefully instantiable class)

Reply:
I think this was a thing we were considering long-term to allow custom types handling which would require some registration system but maybe that'd still be static. I'll change it to static at least for now until we change things later.